units/systemd-timesyncd.service.in

```diff
@@ -12,7 +12,8 @@ ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 DefaultDependencies=no
 RequiresMountsFor=/var/lib/systemd/clock
-After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service
+Requires=dev-rtc.device
+After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service dev-rtc.device
```

Comment:
adjtimex will only result in a sync to RTC if the "11-minute-mode" is enabled, which afaics won't happen if you only call it once. But I figure that's something to investigate. 

(But yeah, if a single adjtimex call with the right params in fact does sync to RTC, then this would indeed be much preferrable over direct /dev/rtc access)